add additional detail about alert rules (#4000)
influxdata/docs-v2: content/chronograf/v1.9/guides/create-alert-rules.md

7. Click **Save Rule**.

## Enable and disable alert rules

To enable and disable alerts, click on **{{< icon "alert" "v2">}} Alerting** in the left navigation bar and select **Alert Rules**.

  - To enable an alert rule, locate the alert rule and click the box **Task Enabled**. A blue dot shows the task is enabled. A message will appear letting you know the rule was enabled successfully. 
  - To disable an alert rule, click the box **Task Enabled**. The blue dot will disappear and you will receive a message letting you know the alert was successfully disabled. 

## Delete alert rules

> To delete an alert, click on **{{< icon "alert" "v2">}} Alerting** in the left navigation bar and select **Alert Rules**.

Reviewer: ```suggestion
To delete an alert, click on **{{< icon "alert" "v2">}} Alerting** in the left navigation menu and select **Alert Rules**.
```

Author: I am going to leave it as bar for consistency because that's how it's described in all the chrono docs.

> 1. Locate the alert you want to delete by its name.  
> 2. Hover over the "Task Enabled" box for the specific rule, and a **Delete** button will appear to the right.  

Reviewer: ```suggestion
1. Locate the alert you want to delete, and then hover over the "Task Enabled" box. A **Delete** button appears to the right.  
```

3. Click **Delete** to delete the rule. 

> **NOTE:** Once you delete a rule, it will be removed permanently. This action cannot be undone.

Reviewer: ```suggestion
**Note:** Deleting a rule cannot be undone, and removes the rule permanently. 
```

Reviewer: ...wonder if we can simply say "Deleting a rule cannot be undone"? seems that implies that it's permanent?
